package/gluon-setup-mode/files/usr/bin/gluon-enter-setup-mode

```diff
@@ -1,6 +1,6 @@
 #!/bin/sh
 
-set -euo pipefail
+set -eu
```

Comment:
this change does a bit more than cosmetics, could you argue why we need to do this and how we make sure this doesn't break?

Reply:
posix-sh does not support the option pipefail.
bash does, dash does, sh does not.

I think it currently does nothing but make you believe the script would respect pipefail, while it does not.

If we want pipefail-like behavior I could implement that as well.